test/cowboy_SUITE.erl

```diff
@@ -284,7 +285,6 @@ stop_cowboy() ->
 execute_request(Host, Path, Method, Headers, Body) ->
     {ok, Pid} = fusco:start_link(Host, []),
     Response = fusco:request(Pid, Path, Method, Headers, Body, 5000),
```

Comment:
A bit reduntant here, instead of capturing the return value and then return it, we could just, return it! :P